### PR TITLE
Websites Maintenance

### LINKED ISSUE #4154: [Comicweb Newtype] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
tried to download ch 14 and 15 which is free to read but unable to download

**To Reproduce**
1. comicweb newtype
2. https://comic.webnewtype.com/contents/witchs_marriage/
3. ch 14 and 15
4. 
eeeeee

**HakuNeko (please complete the following information):**
 - Version [e.g. 6.17] @9436d8

### LINKED ISSUE #4165: [WeLoveManga] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
The connector is broken (mangalist, preview and download).

**HakuNeko (please complete the following information):**
 - nightly and stable

### LINKED ISSUE #4176: [Asura Scans] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
Error when trying to download, Error message "There was no content found for "Chapter ##"  Make sure it's accessible"

**To Reproduce**
1. Asura Scans (Others work)
2. Doctor's Rebirth (Or any other)
3. Chapter 62 (Or any other)
4. Copy any relevant error message that appears in a popup or (advanced) within the console (press F12)
  "There was no content found for "Chapter ##"  Make sure it's accessible"

**Screenshots**
If complex to describe, add screenshots to help explain your problem.

**HakuNeko (please complete the following information):**
 - Version [6.17]

### LINKED ISSUE #4180: Reaperscans connector is not working (no updates)
It's almost 2 weeks since the reaperscans connector was not updated. as the following pic shows that the manhua "Overgeared" updated on the website but not in the ap
Screenshot (73)
Screenshot (72)
p.

### LINKED ISSUE #4181: [team1x1] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
Change the domen they change there domen from 
https://www.team1x1.com
To
https://teamxmanga.com

**To Reproduce**
1. Team x

### PR DESCRIPTION
**Pull request by Sheepux**
Fixes #4180 
Fixes #4181
Fixes #4176
Fixes #4165 (partial : only weloma)
Fixes #4154

### CODE REVIEW

File: src/web/mjs/connectors/ReaperScans.mjs [hunk 1]
```diff
@@ -6,10 +6,34 @@ export default class ReaperScans extends WordPressMadara {
...
+            const promises = await Promise.allSettled([
+                this._getChaptersAjax(manga.id),
+                this._getChaptersAjaxOld(placeholder.dataset.id)
+            ]);
+            data = promises.find(promise => /fulfilled/i.test(promise.status)).value;
+        }
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
+                title: element.querySelector('p').innerText.replace(manga.title, '').trim(),
```
Comment (Sheepux) [inline src/web/mjs/connectors/ReaperScans.mjs]: _getChapter modification not related to the fix: chapter title improvment
adding the querySelector('p') to remove the "least released date" span in the chapter title (didn't want to modify the madara template if this is not happening on another connector).

**Comment by 27136773** [comment]
Guess i was waiting to long...
https://github.com/ronny1982/hakuneko/commits/website-maintenance

**Comment by Sheepux** [comment]
Make sure to assign yourself to the issue first ;)

File: src/web/mjs/connectors/ReaperScans.mjs [hunk 2]
```diff
@@ -6,10 +6,34 @@ export default class ReaperScans extends WordPressMadara {
...
-        this.tags = [ 'webtoon', 'english' ];
+        this.tags = ['webtoon', 'english'];
         this.url = 'https://reaperscans.com';
         this.links = {
             login: 'https://reaperscans.com/login'
         };
+        this.queryChapters = 'li.wp-manga-chapter a';
     }
+
+    async _getChapters(manga) {
```
Comment (27136773) [inline src/web/mjs/connectors/ReaperScans.mjs]: May consider to add a bloat removal to `WordPressMadara` as done in `WordPressMangastream`
e.g. ronny1982@29e3e8a
And then simply define the bloat removal query in `ReaperScans`
e.g. ronny1982@d8468ee

**Review by 27136773 — APPROVED** [review]

**Pull request merged**